usaspending_api/references/account_helpers.py

```diff
@@ -0,0 +1,89 @@
+from django.db import connection
+from usaspending_api.common.etl import ETLQuery, ETLTable
+from usaspending_api.common.etl.operations import delete_obsolete_rows, insert_missing_rows, update_changed_rows
+
+
+# This is basically the desired final state of the federal_account table.  We can diff this against the
+# actual federal_account table and make corrections as appropriate to bring the federal_account table
+# into line.  Since the treasury_appropriation_account and federal_account tables are fairly small, we
+# can perform full diffs with no noticeable performance impact.  This sort order is dictated by DEV-3495.
+FEDERAL_ACCOUNTS_FROM_TREASURY_ACCOUNTS_SQL = """
+    select
+        distinct on (agency_id, main_account_code)
+        agency_id as agency_identifier,
+        main_account_code,
+        concat(agency_id, '-', main_account_code) as federal_account_code,
+        account_title
+    from
+        treasury_appropriation_account
+    order by
+        agency_id,
+        main_account_code,
```

Comment:
Why are these first in ordering? Doesn't this defeat the point of the rest of the ticket?

Reply:
It's a requirement of `distinct on`.

https://www.postgresql.org/docs/10/sql-select.html#SQL-DISTINCT

> The DISTINCT ON expression(s) must match the leftmost ORDER BY expression(s).

How does that defeat the point of the rest of the ticket?

Reply:
Ok, that clears that up.